pyproject.toml

```diff
@@ -11,7 +11,7 @@ repository = "https://github.com/pureskillgg/csgo-dsdk"
 [tool.poetry.dependencies]
 python = "^3.9"
 pandas = "^1.3.2"
-pureskillgg-dsdk = "^1.0.3"
+pureskillgg-dsdk = "1.2.0"
```

Comment:
This should use the carrot `^1.2.0`

Reply:
TY. raised in https://github.com/pureskillgg/csgo-dsdk/issues/8